src/start/RealmScreen.js

```diff
@@ -63,6 +63,7 @@ class RealmScreen extends PureComponent<Props, State> {
     const { styles } = this.context;
     const { initialRealm, navigation } = this.props;
     const { progress, error, realm } = this.state;
+    const isButtonDisable = !realm || realm.length === 0;
```

Comment:
That is not needed. Both `undefined` and `''` are handled already with `!realm`

Reply:
Done 👍 

Sorry I missed it before :)